src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.html

```diff
@@ -238,7 +242,11 @@
                 [buttonLoading]="!!exercise.loading"
                 [smallButton]="smallButtons"
                 [hideLabelMobile]="false"
-                *ngIf="practiceParticipation?.initializationState === InitializationState.INACTIVE"
+                *ngIf="
```

Comment:
You should be able to also call `isResumeExerciseAvailable()` for this case, just with the other participation.